src/components/super-spawner.js

```diff
@@ -51,6 +62,16 @@ AFRAME.registerComponent("super-spawner", {
     const pos = this.data.useCustomSpawnPosition ? this.data.spawnPosition : this.el.getAttribute("position");
     entity.setAttribute("position", pos);
     this.el.sceneEl.appendChild(entity);
+
+    if (this.data.spawnCooldown > 0) {
+      this.el.setAttribute("visible", false);
+      this.el.setAttribute("scale", { x: 0.0001, y: 0.0001, z: 0.0001 });
```

Comment:
Why do you need to modify the scale?

Reply:
the collision shapes are still there and "clickable", so this is to make it "physically" disappear

Reply:
should probably just remove the interactable class